### ISSUE TITLE
[Enhancement] Ability to use string in place of lashing.

### ISSUE THREAD

**Issue by Samarai** [JayCeeCreates/earlygame#33]
Early-game progression currently involves gathering plant fibre using a Flint Knife in order to create lashing, which is used for crafting primitive flint tools. This is relatively easy and very safe, but I think an alternative option for players who take too long on their first day and lack a knife/foliage to gather fibre would be to use string dropped by spiders for crafting flint tools.

This would also have additional knock-on effects in later-game play. Being able to use string means that, in the event a player is deep underground without a clear path out and lacks a durable-enough tool to dig out, they could feasibly craft primitive tools using only things found underground (flint from gravel, string from spiders/cave spiders/cobwebs), with the exception of sticks. (though possessing an axe/saw and being in an underground mineshaft would provide an ample supply)

In short, this would give primitive tools a double purpose as _improvised_ tools even once a player is far past the stone age. If a player requires a flint and steel but doesn't have iron ingots, they could create a firestarter. If a player needs a ranged attack, but lacks arrows and a way to craft them, they could make a slingshot. Etc, etc. This wouldn't be something that would come up often, of course, but it's something to consider.

Thank you for reading this suggestion, and I'm sorry for how wordy and borderline incomprehensible it is!

**Comment by MissGamerz** [comment]
you could create a datapack for this? it would be pretty simple aswell.....

**Comment by JayCeeCreates** [comment]
Just like @MissGamerz said.

Create a datapack that contains:
`data / earlygame / recipes / slingshot_with_string.json`

And within the .json file:
```
{
    "type": "minecraft:crafting_shaped",
    "pattern": [
        "xzx",
        " y ",
        " x "
    ],
    "key": {
        "x": {
            "tag": "c:sticks"
        },
        "y": {
            "tag": "minecraft:planks"
        },
        "z": {
            "item": "minecraft:string"
        }
    },
    "result": {
        "item": "earlygame:slingshot",
        "count": 1
    }
}
```
Closing this.
